CONTRIBUTING.md

```diff
@@ -25,26 +25,25 @@ guidance below.
 ## Changing the codebase
 
 We are no longer forking and pulling. Instead, we are working off feature branches.
-After you've edited a component, open a PR to merge your feature branch back into `canary`.
+After you've edited a component, open a PR to merge your feature branch back into `master`.
 
 For example, if you wanted to change `cf-buttons`, here's what you'd do:
```

Comment:
I never reference this section so I never noticed this (I always use the "Testing in the documentation site" instructions) but I would just make sure this sentence describes the fact that the steps are changing cf-buttons in Capital Framework, while compiling and viewing changes in the cfgov-refresh app. That's sort of implied but not made clear.